### ISSUE TITLE
Feature request: self-voting

### ISSUE THREAD

**Issue by jfritz** [antoinejaussoin/retro-board#23]
As a retrospective facilitator, I want everyone to be able to self-vote on their issues that they submitted, so that we can implement a traditional dot-voting approach.

Dot-voting example: everyone gets two votes, and they can use it on any item. they can use two votes on one item or vote on two separate items.

**Comment by seif** [comment]
I'd also like this functionality currently just removed the code that stop self-voting https://github.com/antoinejaussoin/retro-board/compare/master...opentable:ot

Once we've used in anger, I will make configurable somehow and PR. 

Would allowing self voting be per session or apply to all session? i.e. have the toggle in session like the summary mode toggle or in advanced options for the board?

@jfritz I've also just removed the limit on number of votes user can submit on a single post-it. Thought about enforcing the limit, but that would be a lot of change as ideally for us instead of the current behaviour of likes/dislikes we would only count likes, 'dislike' button would just remove the like.
